fastlane_bot/helpers/txhelpers.py

```diff
@@ -991,19 +991,22 @@ def _get_layer_one_gas_fee_loop(self, signed_transaction) -> Decimal:
         returns: Decimal
             The total fee (in gas token) for the l1 gas fee
         """
-        return asyncio.get_event_loop().run_until_complete(self._get_layer_one_gas_fee(signed_transaction=signed_transaction))
 
-    async def _get_layer_one_gas_fee(self, signed_transaction) -> Decimal:
+        tasks = self.ConfigObj.GAS_ORACLE_CONTRACT.caller.basefee(), self.ConfigObj.GAS_ORACLE_CONTRACT.caller.l1FeeOverhead(), self.ConfigObj.GAS_ORACLE_CONTRACT.caller.l1FeeScalar()
+        ethereum_base_fee, fixed_overhead, dynamic_overhead = asyncio.get_event_loop().run_until_complete(asyncio.gather(*tasks))
+
+        return self._get_layer_one_gas_fee(signed_transaction=signed_transaction, ethereum_base_fee=ethereum_base_fee, fixed_overhead=fixed_overhead, dynamic_overhead=dynamic_overhead)
```

Comment:
Semantic:
No need to specify each parameter name as part of this function-call.
This syntax is only really needed for input arguments which have default values in the called function.
Using it here makes this function-call statement unnecessarily long, and rather difficult to read.